platform-common/src/main/java/com/softicar/platform/common/core/locale/ILocale.java

```diff
@@ -49,4 +49,35 @@ public interface ILocale {
 	 * @return the date format (never <i>null</i> and not empty)
 	 */
 	String getDateFormat();
+
+	/**
+	 * Returns a human-readable date format that uses abbreviations which
+	 * correspond to the language returned by {@link #getLanguage()}.
+	 *
+	 * @return a localized, human-readable date format (never <i>null</i>)
+	 */
+	String getLocalizedDateFormat();
+
+	/**
+	 * Returns a human-readable time format that uses abbreviations which
+	 * correspond to the language returned by {@link #getLanguage()}.
+	 *
+	 * @return a localized, human-readable time format (never <i>null</i>)
+	 */
+	String getLocalizedTimeFormat();
+
+	/**
+	 * Returns a human-readable daytime format that uses abbreviations which
```

Comment:
i think the term `daytime` is not correct here either refer to `{@link DayTime}` or talk about `date and time format` or `format for a point in time`, etc